mappings/net/minecraft/block/Block.mapping

```diff
@@ -1,19 +1,30 @@
 CLASS none/akw net/minecraft/block/Block
 	FIELD A stateFactory Lnone/ask;
 	FIELD a id Lnone/kp;
+	FIELD b creativeTab Lnone/adq;
 	FIELD c defaultState Lnone/asj;
+	FIELD d unlocalizedName Ljava/lang/String;
 	FIELD h registry Lnone/cs;
+	FIELD j FULL_BLOCK_BOX Lnone/bcp;
+	FIELD k NULL_BOX Lnone/bcp;
 	FIELD x material Lnone/ayo;
 	METHOD <init> (Lnone/ayo;Lnone/ayp;)V
 		ARG 0 material
+	METHOD a getUnlocalizedName ()Ljava/lang/String;
 	METHOD a deserializeState (I)Lnone/asj;
 		ARG 0 meta
 	METHOD a register (ILjava/lang/String;Lnone/akw;)V
 	METHOD a register (ILnone/kp;Lnone/akw;)V
+	METHOD a setCreativeTab (Lnone/adq;)Lnone/akw;
+		ARG 0 creativeTab
 	METHOD a getBlockFromItem (Lnone/aeo;)Lnone/akw;
+		ARG 0 item
+	METHOD a addSubItems (Lnone/aeo;Lnone/adq;Lnone/dc;)V
+	METHOD a activate (Lnone/aiu;Lnone/cn;Lnone/asj;Lnone/aak;Lnone/qz;Lnone/cu;FFF)Z
```

Comment:
Either is fine however I prefer more concise names and `onBlockActivated` or `onActivated` doesn't convey any more information than `activate` does.